pkg/flowaggregator/collector/ipfix.go

```diff
@@ -70,49 +89,119 @@ func NewIPFIXCollector(
 			IsEncrypted:   false,
 		}
 	default:
-		return nil, fmt.Errorf("unsupported protocol: %s", aggregatorTransportProtocol)
+		return fmt.Errorf("unsupported protocol: %s", c.aggregatorTransportProtocol)
 	}
 	// Tell the collector to accept IEs which are not part of the IPFIX registry (hardcoded in
 	// the go-ipfix library). The preprocessor will take care of removing these elements.
 	cpInput.DecodingMode = ipfixcollector.DecodingModeLenientKeepUnknown
 	collectingProcess, err := ipfixcollector.InitCollectingProcess(cpInput)
 	if err != nil {
-		return nil, fmt.Errorf("failed to initialize IPFIX collector: %w", err)
+		return fmt.Errorf("failed to initialize IPFIX collector: %w", err)
 	}
 
-	preprocessor, err := newPreprocessor(collectingProcess.GetMsgChan(), recordCh)
+	preprocessor, err := newPreprocessor(collectingProcess.GetMsgChan(), c.recordCh)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create IPFIX preprocessor: %w", err)
+		return fmt.Errorf("failed to create IPFIX preprocessor: %w", err)
 	}
 
-	return &ipfixCollector{
-		collectingProcess: collectingProcess,
-		preprocessor:      preprocessor,
-	}, nil
+	c.collectingProcess = collectingProcess
+	c.preprocessor = preprocessor
+
+	return nil
+}
+
+func (c *ipfixCollector) UpdateCerts() {
+	c.restartCh <- struct{}{}
```

Comment:
The go-ipfix library should be updated to take a standard `tls.Config` as an input parameter, this is too hacky IMO

Reply:
I just hate to make the logic more complex to accommodate for DTLS specifically, but I guess we can consider removing DTLS at a later time